src/python/ensembl/io/genomio/genbank/get_genbank.py

```diff
@@ -15,13 +15,14 @@
 # limitations under the License.
 
 """Download a Genbank file from NCBI from an accession.
-A file is created in the current working dir, with the name {accession}.gb
 
 Raises:
     DownloadError: if the download fails
 """
 
 
+from pathlib import Path
+from os import PathLike
```

Comment:
```suggestion
from os import PathLike
from pathlib import Path

```
And do you mind sorting alphabetically the next two as well? Cheers!

Reply:
sure